AB_Hands_On/AB_Hands_On.ipynb

```diff
@@ -0,0 +1,6985 @@
+{
```

Comment:
<p>In the intro perhaps also state this will be a reference section that you dont need now, but can come back to anytime you see an inference data object later in the course are confused</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='90'/>

Reply:
<p>Good point, will do</p>